includes/class-newspack-ads-model.php

```diff
@@ -676,9 +676,9 @@ function( $size ) {
 
 			// If there is a multisize that's equal to the width and height of the container, remove it from the multisizes.
 			// The container size is included by default, and should not also be included in the multisize.
-			$container_multisize          = $width . 'x' . $height;
-			$container_multisize_location = array_search( $container_multisize, $multisizes );
-			if ( false !== $container_multisize_location ) {
+			$container_multisize           = $width . 'x' . $height;
+			$container_multisize_locations = array_keys( $multisizes, $container_multisize );
+			foreach ( $container_multisize_locations as $container_multisize_location ) {
```

Comment:
Although it's uncommon, there can be stored ad units with duplicate sizes. This ensures that the duplicate size does not get configured into the AMP ad tag multi-size attribute.